docs/connectors/sqlserver.asciidoc

```diff
@@ -410,11 +410,11 @@ The following configuration properties are _required_ unless a default value is
 
 |`table.whitelist`
 |
-|An optional comma-separated list of regular expressions that match fully-qualified table identifiers for tables to be monitored; any table not included in the whitelist will be excluded from monitoring. Each identifier is of the form _schemaName_._tableName_. By default the connector will monitor every non-system table in each monitored schema. May not be used with `table.blacklist`.
+|An optional comma-separated list of regular expressions that match fully-qualified table identifiers for tables to be monitored; any table not included in the whitelist will be excluded from monitoring. Each identifier is of the form _tableName_._schemaName_._tableName_. By default the connector will monitor every non-system table in each monitored schema. May not be used with `table.blacklist`.
```

Comment:
According to https://github.com/debezium/debezium/blob/master/debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java it should be probably:

_databaseName_._schemaName_._tableName_

Reply:
Yeah, thanks. Fixed